index.html

```diff
@@ -4292,6 +4306,446 @@ <h3>Proof Formats</h3>
+(string <var>inputMediaType</var>) as inputs, and produces an object
+(<var>securingMechanismVerified</var>) that contains the following:
+        </p>
+
+        <ul>
+          <li>
+a verification status (boolean <var>status</var>)
+          </li>
+          <li>
+a verified document (object <var>document</var>)
+          </li>
+          <li>
+a media type (string <var>mediaType</var>)
+          </li>
+          <li>
+zero or more warnings (array of <a>ProblemDetails</a> <var>warnings</var>)
+          </li>
+          <li>
+zero or more errors (array of <a>ProblemDetails</a> <var>errors</var>)
+          </li>
+        </ul>
+
+        <p>
+The securing mechanism verification algorithm is as follows:
+        </p>
+
+        <ol class="algorithm">
+          <li>
+Set <var>securingMechanismVerified</var> to an empty object and initialize the
+following fields: <var>status</var> to `false`, <var>warnings</var> to an
+empty array, and <var>errors</var> to an empty array.
+          </li>
+          <li>
+Determine the specific mechanism for extracting the <a>conforming document</a>
+(<var>extractDocument</var>):
+            <ol class="algorithm">
+              <li>
+If the <var>inputMediaType</var> is `application/vc+ld+json` or
+`application/vp+ld+json`, the <var>extractDocument</var> function takes
+<var>inputBytes</var> as an input parameter, parses the value as JSON,
+and returns the result as an object. If JSON parsing fails, a
+<a href="#PARSING_ERROR">PARSING_ERROR</a> is appended to
+<var>securingMechanismVerified</var>.<var>errors</var>.
+              </li>
+              <li>
+If the <var>inputMediaType</var> is associated with the JOSE or COSE
+specifications, the [[?VC-JOSE-COSE]] specification defines a process for
+extracting the <a>conforming document</a>.
+              </li>
+              <li>
+Other extracting functions are possible, but need not be listed in
+the <a data-cite="?VC-SPECS#media-type-extensions">Media Type Extensions</a>
+section of the Verifiable Credentials Specifications Directory [[?VC-SPECS]].
+Implementers might find these other specifications through other standards
+bodies or communities that utilize this specification.
+              </li>
+            </ol>
+          </li>
+          <li>
+Determine the specific cryptographic verification algorithm
+(<var>verifyProof</var>) required to verify the securing mechanism:
+            <ol class="algorithm">
+              <li>
+If the <var>inputMediaType</var> is `application/vc+ld+json` or
+`application/vp+ld+json`, the Verifiable Credential Data Integrity
+Specification [[?VC-DATA-INTEGRITY]] defines the process for identifying
+a specific cryptography suite type used for cryptographic verification by
+utilizing the `proof`.`type` property. The list of all registered
+proof types can be found in the <a data-cite="?VC-SPECS#proof">Proof Types</a>
+section of the Verifiable Credentials Specifications Directory [[?VC-SPECS]].
```

Comment:
Boringly procedural comment... [[VC-SPECS]] is currently an editors' draft, which has not yet been published as a WD. Per W3C pubrules, if the VCDM is to be published as a CR, the VC-SPECS document _MUST_ be at least on a a WD level (if it is a rec-track document, otherwise a draft note).

Reply:
A "boring procedural comment" that is going to prevent publication of VCDM v2.0 as a CR. :)

I believe VC-SPECS is expected to be a Draft NOTE? Some would object to it becoming a registry (although, we should revisit this to see how strong that objection is going to be).

Reply:
I think that should be discussed on the WG call asap. cc @brentzundel

Reply:
Yes, although since DI just went into CR last week... this might be a breaking change that would kick off another CR. See https://github.com/w3c/vc-data-model/pull/1338/files#r1405482587 for more commentary on that concern.

Reply:
I think its best we minimize the unnecessary text, or this PR will never land.

I suggest moving all references to note / informative elaboration to issues, so we can see what is actually normatively changing with this PR.

Reply:
@OR13 wrote:
> I suggest moving all references to note / informative elaboration to issues, so we can see what is actually normatively changing with this PR.

Please make concrete change suggestions for the text that you would like to see removed. The text in there now is a result of weeks of discussion with others in the WG. All of it is there because there were requests for the text.

References to VC-SPECS is there because that is the mechanism that we suggest people use to discover existing Securing Mechanisms such that they can match media types to securing mechanisms. Before we had this text, @jyasskin was concerned that we weren't documenting the process, nor the interface, with enough rigor. Now that the text is there, @OR13 and @selfissued are objecting to the references that were added based on requests from @jyasskin.

The three of you need to come to an agreement on a path forward that won't result in one of you, or another member of the WG, objecting when the changes are made, because each of you seem to be asking for conflicting things.